### PR TITLE
Update _fhi-datepicker.block.scss

### PR DESCRIPTION
**Pull request by yleaxeman**
Note: no changelog update due to the fact that the Datepicker is not yet released

### CODE REVIEW

File: src/blocks/_fhi-datepicker.block.scss [hunk 1]
```diff
@@ -5,6 +5,16 @@
         display: block;
     }
 
+    &__description {
+        color: $fhi-grey-3;
+        display: block;
+        margin-bottom: $fhi-space-1;
+
+        .fhi-datepicker__label + & {
```
Comment (proand) [inline src/blocks/_fhi-datepicker.block.scss:13]: Will we ever need more than one `fhi-datepicker__description`-element?

**Review by proand — COMMENTED** [review]
Changelog?

**Comment by yleaxeman** [comment]
> Changelog?

See description

File: src/blocks/_fhi-datepicker.block.scss (hunk 1 above)
Comment (yleaxeman) [reply src/blocks/_fhi-datepicker.block.scss:13]: No, but I've taken into account that the description might not necessarily be following a `fhi-datepicker__label`. The label might be missing

Comment (proand) [reply src/blocks/_fhi-datepicker.block.scss:13]: Aha! I see.

**Pull request merged**